src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/Content/link.html.twig

```diff
@@ -36,9 +36,9 @@
 {% endif %}
 
 {% block link %}
-<a 
```

Comment:
I don't know if this space was needed but now PHPStorm won't let me add it back :sweat_smile: 

If it's needed I can use some other editor to add it back.